pyproject.toml

```diff
@@ -38,3 +38,8 @@ dependencies = [
 
 [project.urls]
 Source = "https://github.com/caikit/caikit"
+
+[tool.pytest.ini_options]
+markers = [
+    "examples: marks tests as e2e examples (deselect with '-m \"not examples\"')",
+]
```

Comment:
A custom test marker is used to make examples e2e tests. 
Example tests are slightly longer running, so this gives an easy way to skip them for local runs.

Reply:
Do you usually run the test on laptops through `tox`?
I can make the tox job to not run the test by default and enable it in the github action only

Reply:
I made the test excluded by default via tox, it's only enabled in the GitHub workflow

Reply:
> Do you usually run the test on laptops through tox?

Yeah, either a simple `tox` for tests + formatting, or a `tox -e py` for just the tests